LondonBritishLibrary/orient/BLorient13156.xml

```diff
@@ -290,12 +356,24 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         </profileDesc>
         <revisionDesc>
             <change who="JK" when="2023-02-14">Created entity</change>
-            <change when="2024-04-02" who="CH">Update Contents, additions and provenance.</change>
+            <change when="2024-04-02" who="CH">Update Contents, additions and provenance</change>
+            <change when="2024-07-08" who="CH">Update dimensions, decorations and bibliography</change>
         </revisionDesc>
     </teiHeader>
     <text>
         <body>
-            <div type="bibliography"/><!---->
+            <div type="bibliography">
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Uhlig1988Palaographie"/>
+                        <citedRange unit="page">251-253</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Mercier2000LArcheEthiopienne"/>
```

Comment:
note that Mercier already exists in Zotero as bm:MercierArt2001; you created a doublet, please remove the doublet and update the tag.